evm_loader/program/src/account/mod.rs

```diff
@@ -33,6 +35,9 @@ mod state_finalized;
 pub mod token;
 mod treasury;
 
+pub const HEAP_OFFSET_PTR: usize = holder::HEAP_OFFSET_OFFSET;
+pub const MIN_HEAP_OFFSET: usize = holder::BUFFER_OFFSET;
+
 pub const TAG_EMPTY: u8 = 0;
 pub const TAG_STATE: u8 = 24;
```

Comment:
Please increment this by 1, so it is not possible to read heap from the active holder in the old format